### ISSUE TITLE
In quick start doc, add links to next examples the user might want to look at

### ISSUE THREAD

**Issue by mfisher87** [nsidc/earthaccess#544]
A small sample of examples that the user might be interested in following the initial "quick win" example.

E.g. :

* out of region streaming (cc @cboettig)
* ??

**Comment by asteiker** [comment]
Can we use this existing Issue to characterize the UX feedback from @saberbrasher ?

**Description:**  
The current Quick Start guide is too minimal and leaves users confused at several points. It provides only a small snippet of authentication and dataset search, but skips context. Specific issues noted:  

- Example 2 does not clarify what an Earthdata Login "Auth ID" is or where to find it (good place to add a definition and a link to a glossary entry?).  
- References to different dataset versions are not addressed, leaving ambiguity for beginners (when I tested this, there were two version of SPL4CMDL live and the code did not default to the newest, for example).  
- The Quick Start duplicates but does not align with content already in the NSIDC "How to Use Earthdata" (<https://nsidc.org/data/user-resources/help-center/downloading-nsidc-daac-earthdata-cloud-data-using-earthaccess>) help article.  

**Proposed Work:**  

- [ ] Expand the Quick Start into a deeper walkthrough that utlizes more function arguments.
- [ ] Include concrete examples with real dataset shortnames and ensure they run end-to-end.  
- [ ] Add notes about dataset versions and how to handle them in search queries.  
- [ ] Link out to the longer tutorials (User Guide), the API, and NASA/NSIDC articles for users who need more background.

**Comment by saberbrasher** [comment]
I was about to start a new issue about Quick Start but will instead just add more to this one from my notes.... 
The Quick Start installation instructions could be improved/ expanded a bit for first time users. 

I propose the following: 
- Give full installation steps for the most common workflows... likely link out to dependable resources for this as opposed to creating new content."New to Python? Here is a step by step guide to get things up and running... link. Once Python is running version 3.8 or later, 
download earthaccess in one of the following ways:" then have mamba, conda, pip, etc. 
- Part of the code in the first code block is not actually something a user runs, but instead is output if they properly installed python. To reduce confusion I think we should only have code that we actually 
want them to run in the box. 
- In the log in, we tell users they can log in manually, but do not give any instructions for/ show code for doing so (and instead link to a page for using a .netrc file). Even if a user comes with python experience, a .netrc file (and logging in this way) may be a new concept to them. 
If they just want to start working without additional set up, logging in manually may be more attractive. 
- In step 2, there is no feedback to let the user know the search worked... something as simple as a print results could do, but assuming they came here with their own product in mind, a quick output (with a sentence or two) that lets them know it worked would be good. 
- A short paragraph after Step 2 that explains the arguments and then links to other resources where they are more comprehensively defined would help users who have a specific product in mind already but don't know all the arguments possible in the search function.

I think these changes would yield a more reliable installation process, less user confusion, and fewer issues from incomplete or failed installs/ downloads.

**Comment by saberbrasher** [comment]
@andypbarrett what are your thoughts on some of these suggestions for Quick Start?
